testing/test-helper.js

```diff
@@ -307,3 +307,37 @@ export class ImagePixelVerifier {
     return this.imagePixels_[this.imagePixels_.length - 1].src;
   }
 }
+
+/**
+ * Returns a promise that fetches the content of the AMP ad at the amp4test url.
+ * This somewhat simulates rendering an ad by getting its content from an ad
+ * server.
+ */
+export function fetchAdContent() {
```

Comment:
once the above 2 tests files merged into one, we can move those test helper methods into that file too, given I don't see other component will be interested on them.

Reply:
Done.